src/components/Stages/StagesController.tsx

```diff
@@ -0,0 +1,205 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
rename the file name to match component name "StageController" (singular domain names)

Reply:
renamed as Stage Controller